fitbenchmarking/utils/fitbm_result.py

```diff
@@ -347,7 +347,9 @@ def norm_acc(self):
             if self.min_accuracy in [np.nan, np.inf]:
                 self._norm_acc = np.inf
             else:
-                self._norm_acc = self.accuracy / self.min_accuracy
+                best = 1e-10 if self.min_accuracy in [0.0, 0]\
+                    else self.min_accuracy
+                self._norm_acc = self.accuracy / best
```

Comment:
Is it worth printing a warning that the rel results are just a scaled version of the abs results when this triggers?